task_16.py

```diff
@@ -0,0 +1,21 @@
+# Задача 16: Требуется вычислить, сколько раз встречается некоторое число X в массиве A[1..N]. Пользователь
+# в первой строке вводит натуральное число N – количество элементов в массиве. В последующих строках
+# записаны N целых чисел Ai. Последняя строка содержит число X
+# n = 5
+# 1 2 3 4 5
+# x = 3
+
+# -> 1
+
+N = abs(int(input('Введите количество элементов списка А: ')))
+A_entered = input("Введите через пробел элементы списка: ").split()
+A_num = list(map(int, A_entered))
+if len(A_num) != N:
+    print('Введенные элементы не соответствуют заявленному количеству!')
+else:
+    X = int(input('Введите число X, которое необходимо найти в списке: '))
+    count = 0
+    for i in range(N):
+        if A_num[i] == X:
```

Comment:
Можно проходиться напрямую по элементам списка.
```suggestion
    for i in A_num:
        if i == X:
```